core/tauri/scripts/toggle-devtools.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+(function () {
+  function toggleDevtoolsHotkey() {
+    const isHotkey = navigator.appVersion.includes("Mac")
+      ? (event) => event.metaKey && event.altKey && event.key === "I"
+      : (event) => event.ctrlKey && event.shiftKey && event.key === "I";
+
+    document.addEventListener("keydown", (event) => {
+      if (isHotkey(event)) {
+        window.__TAURI_INVOKE__("plugin:window|internal_toggle_devtools");
```

Comment:
This won't work since you copied the command call from v2. Remember to always test the change :P

Reply:
I actually tested it but looks like it worked because I am on Windows which like a normal browser have this shortcut by default/